pxb/docker/test-binary-pxb24

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Execute this tool to test binary releases
+#
+#  possible jenkins vars:
+#      CMAKE_BUILD_TYPE = (RelWithDebInfo Debug)
+#      DEFAULT_TESTING = (yes no)
+#      XBTR_ARGS
+#      XTRABACKUP_TARGET
+#      XTRABACKUP_TARGET_VERSION
+
+set -o errexit
+set -o xtrace
+WORKSPACE=$(
+  cd ${1:-./build/pxb24}
+  pwd -P
+)
+
+rm -fr ${WORKSPACE}/PXB
+mkdir -p ${WORKSPACE}/PXB
+tar -C ${WORKSPACE}/PXB --strip-components=1 -zxpf $(ls $WORKSPACE/pxb24.tar.gz | head -1)
+
+# CentOS 6 & 7
+if [[ -f /opt/rh/devtoolset-7/enable ]]; then
+  source /opt/rh/devtoolset-7/enable
+fi
+
+if [[ -f /usr/bin/apt-get ]]; then
+    DIST=$(lsb_release -sc)
+    if [[ "$DIST" == 'focal' ]]; then
+        sudo apt install -y python3-sphinx
+    else
+        sudo apt install -y python-sphinx
+    fi
+fi
+
+XBTR_ARGS+=" "
+
+status=0
+#
+# Running XBTR test cases
+SSL_VER=$(ls -la {/,/usr/}{lib64,lib,lib/x86_64-linux-gnu}/libssl.so.1.* | sed 's/.*[.]so//; s/[^0-9]//g' | head -1)
+echo "Running XBTR suite: $suite"
+TEST_DIR=$(dirname "$(find ${WORKSPACE}/PXB -name bootstrap.sh)")
+cd ${TEST_DIR}
+
+mkdir -p server-tarball/${XTRABACKUP_TARGET}
+if [[ ${XTRABACKUP_TARGET} == "galera57" ]]; then
+    wget --progress=dot:giga \
+    -O server-tarball/${XTRABACKUP_TARGET}/Percona-XtraDB-Cluster-5.7.31-rel34-43.2.Linux.x86_64.glibc2.12-minimal.tar.gz \
+    https://www.percona.com/downloads/Percona-XtraDB-Cluster-57/Percona-XtraDB-Cluster-5.7.31-31.45-2/binary/tarball/Percona-XtraDB-Cluster-5.7.31-rel34-43.2.Linux.x86_64.glibc2.12-minimal.tar.gz
+    rm -rf server
+    mkdir server
+    tar xfz server-tarball/${XTRABACKUP_TARGET}/Percona-XtraDB-Cluster-5.7.31-rel34-43.2.Linux.x86_64.glibc2.12-minimal.tar.gz -C server
+    mv server/*/* server/
+elif [[ ${XTRABACKUP_TARGET} == "galera56" ]]; then
+    wget --progress=dot:giga \
+      -O server-tarball/${XTRABACKUP_TARGET}/Percona-XtraDB-Cluster-$(uname -m).tar.gz \
+      https://www.percona.com/downloads/Percona-XtraDB-Cluster-56/Percona-XtraDB-Cluster-5.6.49-28.42-2/binary/tarball/Percona-XtraDB-Cluster-5.6.49-rel89.0-28.42.2.Linux.$(uname -m).ssl${SSL_VER}.tar.gz
+    rm -rf server
+    mkdir server
+    tar xfz server-tarball/${XTRABACKUP_TARGET}/Percona-XtraDB-Cluster-$(uname -m).tar.gz -C server
+    mv server/*/* server/
+else
+    sudo rm -rf bootstrap.sh
```

Comment:
Please remove `rm -rf` of bootstrap and make it like: https://github.com/Percona-Lab/jenkins-pipelines/commit/2669588e2ea26a6756e8e4bb34a3998523b33198

Reply:
done